### DISCUSSION TITLE
Details a bit the README

### DISCUSSION THREAD

**Discussion by DGrv** [danielgatis/rembg#447] [General]
Hi,

I took a while to understand the help to get extra parameters that I was looking for:

`rembg --help`

> Usage: rembg [OPTIONS] COMMAND [ARGS]...
>
>Options:
>  --version  Show the version and exit.
 > --help     Show this message and exit.
>
>Commands:
>  i  for a file as input
>  p  for a folder as input
>  s  for a http server

I was actually looking for this and I think it would be an idea to make it a bit clear in the first command to reach this.
And do you have **some example to use the different alpha parameters** ? it is not quite clear for me.
Thanks :)

`rembg i --help`

> Usage: rembg i [OPTIONS] INPUT OUTPUT
> 
>   for a file as input
> 
> Options:
>   -m, --model [u2net|u2netp|u2net_human_seg|u2net_cloth_seg|silueta]
>                                   model name  [default: u2net]
>   -a, --alpha-matting             use alpha matting
>   -af, --alpha-matting-foreground-threshold INTEGER
>                                   trimap fg threshold  [default: 240]
>   -ab, --alpha-matting-background-threshold INTEGER
>                                   trimap bg threshold  [default: 10]
>   -ae, --alpha-matting-erode-size INTEGER
>                                   erode size  [default: 10]
>   -om, --only-mask                output only the mask
>   -ppm, --post-process-mask       post process the mask
>   --help                          Show this message and exit.

**Comment by renato-bosa** [comment]
Yes, I'm looking for some information about **alpha parameters** too.

Thanks.
